xcm/pallet-xcm-benchmarks/src/lib.rs

```diff
@@ -64,7 +62,7 @@ pub fn mock_worst_case_holding() -> MultiAssets {
 	const HOLDING_FUNGIBLES: u32 = 99;
 	const HOLDING_NON_FUNGIBLES: u32 = 99;
 	let fungibles_amount: u128 = 100;
-	(0..HOLDING_FUNGIBLES)
+	(1..=HOLDING_FUNGIBLES)
```

Comment:
This is done here because the XCM executor automatically removes any fungible assets that have an amount of 0, which would cause some tests to fail as the worst case holding expects a non-existent `GeneralIndex(0)` asset to be there.